libraries/WiFiProv/src/WiFiProv.cpp

```diff
@@ -17,6 +17,10 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
     
 */
+#pragma once
```

Comment:
#pragma once is necessary in a cpp file?
```suggestion
```

Reply:
it seems to be a copy&paste result.

Reply:
funny... I do not remember touching this file at all 😆 maybe it felt left out and decided to make a statement